components/inlinealert/index.css

```diff
@@ -38,9 +38,11 @@ governing permissions and limitations under the License.
 }
 
 @media (forced-colors: active) {
-  --highcontrast-inlinealert-background-color: Background;
-  --highcontrast-inlinealert-header-and-content-color: CanvasText;
-  --highcontrast-inlinealert-border-and-icon-color: ButtonBorder;
+  .spectrum-InLineAlert {
+    --highcontrast-inlinealert-background-color: Background;
+    --highcontrast-inlinealert-header-and-content-color: CanvasText;
+    --highcontrast-inlinealert-border-and-icon-color: ButtonBorder;
+  }
```

Comment:
@yosevu I missed this in my previous review, so I made the change myself. I think we should try to stick with scoping the `highcontrast` custom properties to the component's selector. Previously, (without the selector) these custom properties were just being defined without any selector scope.